src/utils/user.controller.js

```diff
@@ -0,0 +1,56 @@
+const HTTPStatus = require('http-status');
+
+const { generateToken } = require('./auth');
+
+function login({ model, body: { email, password } }) {
```

Comment:
Did ya test these? Coz pretty sure they gonna break - the params need to be added to the inner function. The other controller has an outer function to pass the name of the model in

Reply:
I haven't tested it because it isn't really in a working state. I just moved the code across to a separate file and added the extra endpoints.